package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "nestjs-graphile-worker",
-  "version": "0.3.2",
+  "version": "0.3.3",
```

If you can't manage to use 9.x and 10.x at the same time (see comment below), could you increase the version as 
```suggestion
  "version": "0.4.0",
```